package.json

```diff
@@ -49,5 +51,11 @@
     "Nate Rajlich <nathan@tootallnate.net> (https://github.com/tootallnate)",
     "Sean McArthur <sean.monstar@gmail.com> (https://github.com/seanmonstar)",
     "Fanie Oosthuysen <fanie.oosthuysen@gmail.com> (https://github.com/weareu)"
-  ]
+  ],
+  "binary": {
+    "module_name": "bcrypt_lib",
+    "module_path": "./lib/binding/",
+    "host": "https://github.com",
+    "remote_path": "/ksmyth/node.bcrypt.js/releases/download/v{version}/"
```

Comment:
`remote_path` substitutes in the current version number, so you don't need to change the URL.

That sounds like a reasonable workflow.

I haven't published this to npm. The workflow I use is to `npm install [full URL to the release .tgz]` after uploading the binaries. Then `npm install` to install everything else.